bcgov/HOUS-permit-portal
# Bugfix/bphh 933/submitter radio input and other fixes

## Description
- [Re-add rich text for requirement help text removed by error in previous merge](https://github.com/bcgov/HOUS-permit-portal/commit/3fc0892890ed0efd802b61c2009222930906e1d5)
- [Sanitize rich text attributes](https://github.com/bcgov/HOUS-permit-portal/commit/e83a8c3c9af00f9e7915676d5c156d9824adf109)
- [Fix help text for checkbox requirements](https://github.com/bcgov/HOUS-permit-portal/commit/36aa99e8acf371bafbe4bd93fff5f290b72559f0)
- [Fix radio requirement form json](https://github.com/bcgov/HOUS-permit-portal/commit/3b51dd6b9cc82b492a0794a9d51b72747e512232)


## What type of PR is this? (check all applicable)

- [x ] 🐛 Bug Fix

## Related Tickets & Documents
- https://hous-bssb.atlassian.net/browse/BPHH-983


## Steps to QA
1. [Re-add rich text for requirement help text removed by error in previous merge](https://github.com/bcgov/HOUS-permit-portal/commit/3fc0892890ed0efd802b61c2009222930906e1d5)
    -  Log in as super admin and navigate to requirements library
    - Select a block to edit
    - Select any requirement to edit
    - Should be able to add rich text to help Text
2. [Fix help text for checkbox requirements](https://github.com/bcgov/HOUS-permit-portal/commit/36aa99e8acf371bafbe4bd93fff5f290b72559f0)
    - Log in as super admin and navigate to requirements library
    - Select a block to edit
    - Select or add a checkbox requirement to edit
    - Should be able to add help text
3. [Fix radio requirement form json](https://github.com/bcgov/HOUS-permit-portal/commit/3b51dd6b9cc82b492a0794a9d51b72747e512232)
      - Pre req: ensure a published template  version has radio option
      - Login as submitter and create application with above template
      - The radio requirement should be visible and persist on save
